beatport_importer.user.js

```diff
@@ -131,6 +143,16 @@ function insertLink(release, release_url) {
         )}${MBImport.buildSearchButton(release)}</li>`
     ).hide();
 
+    $(
+        '<form class="musicbrainz_import"><button type="submit" title="Submit ISRCs to MusicBrainz with kepstin’s MagicISRC"><span>Submit ISRCs</span></button></form>'
+    )
+        .on('click', event => {
+            const query = isrcs.map((isrc, index) => (isrc == null ? `isrc${index + 1}=` : `isrc${index + 1}=${isrc}`)).join('&');
```

Comment:
```suggestion
            const query = isrcs.map({ isrc, number } => (isrc == null ? `isrc${number}=` : `isrc${number}=${isrc}`)).join('&');
```
2b. Use the track number instead of the index.